shared/src/price_estimation/gas.rs

```diff
@@ -0,0 +1,24 @@
+///! constants to estimate gas use in GPv2
+
+/// minimum gas every settlement takes
+pub const SETTLEMENT: u64 =
+    // initial tx gas
+    32_000 +
+    // isSolver
+    7365;
+
+/// gas per trade excluding c20 transfer
+pub const TRADE: u64 =
+    // computeTradeExecutions
+    35_000 +
+    // transferFromAccounts and transferToAccount overhead
+    2 * 3000 +
+    // overhead of one interaction
+    3000;
+
+/// lower bound for an erc20 transfer
+/// TODO: Use an average or median?
+pub const ERC20_TRANSFER: u64 = 25_551;
```

Comment:
Nit, I think the cheapest ERC_TRANSFER is 20k (e.g. https://dashboard.tenderly.co/tx/main/0xab9c670bf129a70361f09ed47c503b7c2be6d02a32ecb5fe4a617bdd563e94e5/gas-usage)

Reply:
This was changed to be the median and not the minimum. I think the comment may have been merged out of date. See: https://github.com/gnosis/gp-v2-services/pull/1334#discussion_r740232223